.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -211,17 +229,18 @@ jobs:
           repo: broadinstitute/terra-github-workflows
           ref: refs/heads/main
           token: ${{ env.TOKEN }}
-          inputs: '{
-            "additional-args": "{\"logging\":\"true\",\"java-version\":\"17\",\"billing-project\":\"${{ needs.params-gen.outputs.project-name }}\"}",
-            "run-name": "${{ env.RAWLS_SWAT_TESTS_RUN_NAME }}",
-            "bee-name": "${{ env.BEE_NAME }}",
-            "ENV": "qa",
-            "ref": "refs/heads/${{ needs.init-github-context.outputs.branch }}",
-            "test-group-name": "workspaces_azure",
-            "test-command": "${{ env.rawls_test_command }}",
-            "e2e-env": "${{ env.E2E_ENV }}",
-            "user-subjects": ${{ toJson(env.USER_SUBJECTS) }}
-          }'
+          inputs: |
+            {
+              "additional-args": "{\"logging\":\"true\",\"java-version\":\"17\",\"billing-project\":\"${{ needs.params-gen.outputs.project-name }}\"}",
+              "run-name": "${{ env.RAWLS_SWAT_TESTS_RUN_NAME }}",
+              "bee-name": "${{ env.BEE_NAME }}",
+              "ENV": "qa",
+              "ref": "${{ inputs.build-branch && needs.init-github-context.outputs.fully-specified-branch || '' }}",
```

Comment:
The action (`Rawls-swat-tests`) can handle the empty string (in which case it does not check out a branch).